### ISSUE TITLE
Ubuntu 23.10 (Mantic) Lightzone 5.0.0 Beta 2 Doesn't start

### ISSUE THREAD

**Issue by sfink16** [ktgw0316/LightZone#322]
**Describe the bug**

Ubuntu 23.10 (Mantic) Lightzone 5.0.0 Beta 2 Doesn't start when running under the Ubuntu 22.04 OS.  I can no longer install Lightzone Ubuntu 22.04 (Jammy).  I'm stuck not being able to run either.

**To Reproduce**
1. Install via the --> sudo add-apt-repository ppa:lightzone-team/lightzone <-- PPA
2. Run sudo apt update
3. Run sudo apt install Lightzone
4. The installation is successful to run Lightzone 5.0.0 Beta2.
5. The following error is return:
![Screenshot from 2023-10-10 16-45-14](https://github.com/ktgw0316/LightZone/assets/62555600/7695d3e9-afc7-4ac8-a8f6-164bb1ecca15)

**Expected behavior**
Lightzone to start with no errors.

**Screenshots**
![Screenshot from 2023-10-10 16-55-40](https://github.com/ktgw0316/LightZone/assets/62555600/2f38446b-d0f0-40ef-ac58-f4f5d6f11394)

**Desktop (please complete the following information):**
 - Version Lightzone 5.0.0 Beta 2
 - OS: Ubuntu 22.04 (Jammy)

**Additional context**
The installation seems to force a different version (23.10) then my OS (22.04).  I can't figure out why.
![Screenshot from 2023-10-10 16-55-40](https://github.com/ktgw0316/LightZone/assets/62555600/83766b7c-7531-434b-8516-f792b581d474)
![Screenshot from 2023-10-10 16-57-08](https://github.com/ktgw0316/LightZone/assets/62555600/2244eb39-697d-4e44-8b85-bb084611e3b2)
![Screenshot from 2023-10-10 16-56-54](https://github.com/ktgw0316/LightZone/assets/62555600/81e2b9c6-6d00-4607-9a40-a9ce136f7ea4)

**Comment by sfink16** [comment]
Note: I get the same results when I run on an Ubuntu 23.10 Live CD.

**Comment by ktgw0316** [comment]
@sfink16 

> java.lang.ClassNotFoundError: jiconfont/icons/FontAwesome

I think I already solved this error in #295.  In my virtualbox, the `jiconfont.icons.FontAwesome` exists in classpath (`/usr/share/java/*`).

```sh
$ dpkg -l libjiconfont-font-awesome-java | tail -1
ii  libjiconfont-font-awesome-java 4.7.0.0-2    all    jIconFont - Font Awesome
$ jar tf /usr/share/java/jiconfont-font_awesome.jar | grep "FontAwesome"
jiconfont/icons/FontAwesome$1.class
jiconfont/icons/FontAwesome.class
```

Could you check the same commands as above on your machine?

**Comment by sfink16** [comment]
Remember @ktgw0316 , I'm running 23.10 not 23.04 as per issue 295.  I needed to run 23.10 due to CR3 only working in 23.10.  That may be the reason you're getting different results then me.  Here are my results from the two commands that you list above, although it seems like I'm doing the commands wrong:
![Screenshot from 2023-10-14 22-31-08](https://github.com/ktgw0316/LightZone/assets/62555600/14134849-c019-41b8-bcfc-466fc3b28ca2)

**Comment by ktgw0316** [comment]
> I'm running 23.10 not 23.04 as per issue 295. I needed to run 23.10 due to CR3 only working in 23.10.

Yes, I understand that. I also updated the virtualbox to 23.10, but maybe there is something not cleanly updated on my side. My `libjiconfont-font-awesome-java` version is different than yours, so I'll look into it.

**Comment by sfink16** [comment]
@ktgw0316  - I messed up my screenshot by not installing the II that was recommended.  Maybe this is better:
![Screenshot from 2023-10-14 22-31-08](https://github.com/ktgw0316/LightZone/assets/62555600/e80be92b-24e1-4ac1-b2df-24eaa4b7aec0)

**Comment by ktgw0316** [comment]
I upgraded the `libjiconfont-font-awesome-java` to 4.7.0.1-1, and now lightzone crashes as you said. Sorry about that. I need to fix this issue not only for mantic but lunar and kinetic.

The difference is:
* 4.7.0.0: jiconfont/icons/FontAwesome.class
* 4.7.0.1: jiconfont/icons/font_awesome/FontAwesome.class

**Comment by sfink16** [comment]
Thanks!  At least I know a problem does exist and it wasn't only me that had the problem.

**Comment by ktgw0316** [comment]
Source code change to fix this issue is trivial, but applying it only certain version of distribution when building packages on launchpad PPA is little bit tough...

**Comment by sfink16** [comment]
It sounds complicate considering 3 different versions.  If I can help in any way, let me know, although I haven't coded since my COBOL mainframe days many years ago.

**Comment by ktgw0316** [comment]
Note for myself:

| fontawesome ver. | Launchpad recipe | comment |
| -- | -- | -- |
| 4.7.0.0 | [lightzone-daily](https://code.launchpad.net/~arctica0316/+recipe/lightzone-daily) ||
| 4.7.0.1 | [lightzone-daily-22.10](https://code.launchpad.net/~arctica0316/+recipe/lightzone-daily-22.10) | patch from [font-awesome-patch branch](https://github.com/ktgw0316/LightZone/compare/master...font-awesome-patch) |

**Comment by ktgw0316** [comment]
Fix released on PPA (`lightzone - 5.0.0~beta2-0~202310150417`).

**Comment by sfink16** [comment]
Success!!! Thanks @ktgw0316 !!  Colors and exposure may be off a little bit, but I'm still testing.  Of course we don't have RTC that Tex used to create but we can create our own if needed and desired.  You have notes on this discussion, so I'll leave it up to you to close if you think it can be closed.  Thanks again!
